### DISCUSSION TITLE
The addition of the OTI-087 (if it's feasible)

### DISCUSSION THREAD

**Discussion by Fenix770** [86Box/86Box#3871] [Ideas]
**Is your feature request related to a problem? Please describe.**

No. It would be interesting to have the OTI-087 card emulated. I know this video card supports 512KB, 1MB and 2MB configurations, but unlike the OTi-077 and earlier, it seems to have 2D acceleration and have launched for 16-bit ISA and VLB ports. According the datasheet the ISA version seems to be compatible with the 8-bit bus. The only drawback is it would be more difficult to add because the mentioned acceleration.

**Describe the solution you'd like**
Implementation of this card.

**Additional context**
Here is VGA museums site with the datasheet and BIOS file. 

http://www.vgamuseum.info/index.php/cards/item/327-oak-oti-087

I also have the driver disk:
[Oti-087.zip](https://github.com/86Box/86Box/files/6370149/Oti-087.zip)

**Comment by OBattler** [comment]
I believe the OTI-087 is accelerated, which is why it hasn't been added yet.
